dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6/src/org/apache/cxf/common/injection/Messages.properties

```diff
@@ -0,0 +1,33 @@
+#
+#
+#    Licensed to the Apache Software Foundation (ASF) under one
+#    or more contributor license agreements. See the NOTICE file
+#    distributed with this work for additional information
+#    regarding copyright ownership. The ASF licenses this file
+#    to you under the Apache License, Version 2.0 (the
+#    "License"); you may not use this file except in compliance
+#    with the License. You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing,
+#    software distributed under the License is distributed on an
+#    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#    KIND, either express or implied. See the License for the
+#    specific language governing permissions and limitations
+#    under the License.
+#
+#
+RESOURCE_RESOLVE_FAILED=failed to resolve resource {0}
+FAILED_TO_INJECT_FIELD=access to field is not allowed
+INJECTION_SETTER_NOT_VISIBLE=setter {0} is not visible
+INJECTION_SETTER_RAISED_EXCEPTION=exception raised when invoking setter {0}
+PRIVATE_FIELD_INJECTION_NYI=resource injection for private fields is not yet implemented: {0}
+INJECTION_COMPLETE_NOT_VISIBLE=method annotated by @PostConstruct is not visible: {0}
+INJECTION_COMPLETE_THREW_EXCEPTION=method annotated by @PostConstruct throws exception when invoked
+PRE_DESTROY_NOT_VISIBLE=Method annotated by @PreDestroy is not visible: {0}
+PRE_DESTROY_THREW_EXCEPTION=Method annotated by @PreDestroy throws exception when invoked
+SETTER_INJECTION_WITH_INCORRECT_TYPE=setter for resource found but with wrong signature: {0}
```

Comment:
**change**
setter for resource found but with wrong signature: {0}
**to**
The setter for the resource was found but with wrong signature: {0}